src/usd/sdf_usd_parser/sensor.cc

```diff
@@ -44,11 +44,14 @@ namespace usd
 
     // TODO(adlarkin) check units to make sure they match (no documented units for SDF)
     usdCamera.CreateFocalLengthAttr().Set(
-        static_cast<float>(sdfCamera->LensFocalLength()));
+        static_cast<float>(sdfCamera->LensFocalLength() * 10));
     usdCamera.CreateClippingRangeAttr().Set(pxr::GfVec2f(
           static_cast<float>(sdfCamera->NearClip()),
           static_cast<float>(sdfCamera->FarClip())));
 
+    usdCamera.CreateHorizontalApertureAttr().Set(
+      static_cast<float>(sdfCamera->HorizontalFov().Degree()));
+
     // TODO(adlarkin) Do I need to handle the following in USD
     // (I don't think there's an SDF equivalent):
     //  * horizontal and vertical aperture (there's also an offset for these)
```

Comment:
```suggestion
    usdCamera.CreateHorizontalApertureAttr().Set(
      static_cast<float>(sdfCamera->HorizontalFov().Degree()));
```
I believe the `TODO` can be removed since you addressed the horizontal aperture attribute.